### ISSUE TITLE
[ICC] Raging Spirit disappears after transition 

### ISSUE THREAD

**Issue by Derolana** [dalaranwow/dalaran-wow#5219]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
3seconds before the transition ends, a raging spirit spawns on our shadow priest and disappears during Quake https://youtu.be/TMt9OpwJirQ?t=3m40s , and then appears again on first Harvest Soul https://youtu.be/TMt9OpwJirQ?t=8m15s
**Expected behaviour**: Tell us what should happen instead
The raging spirit shoudnt dissapear.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by JohnADC** [comment]
Can confirm Raging Spirit bugging on platform, thought never saw it run back. We think it might be connected with line of sight, for example person that is targeted by spirit goes off the ledge before spirit becomes active(we never tested if thats how it actually works, just a wild guess considering it only happened a few times on transitions from ledge to platform)

**Comment by Derolana** [comment]
https://imgur.com/mqGfuuB

**Comment by Lothloryen** [comment]
Hello there,

Thank you very much for this report. I didn't know about this, I will push a fix.

**Comment by Derolana** [comment]
https://streamable.com/cw1tk , so the person said he just jumped after the spirit spawns, he runs backwards doing 1 jump.
